notices/ltc-asset-transfer.vbs

```diff
@@ -56,40 +57,55 @@ changelog_display
 'connecting to MAXIS
 EMConnect ""
-Call MAXIS_case_number_finder(MAXIS_case_number)
+CALL MAXIS_case_number_finder(MAXIS_case_number)    'Grabbing the CASE Number
+call Check_for_MAXIS(false)                         'Ensuring we are not passworded out
 get_county_code
 
-Dialog1 = ""
-BeginDialog Dialog1, 0, 0, 306, 100, "LTC asset transfer dialog"
-  Text 30, 10, 45, 10, "Case number:"
-  EditBox 80, 5, 50, 15, MAXIS_case_number
-  Text 165, 10, 60, 10, "ER date (MM/YY):"
-  EditBox 230, 5, 20, 15, renewal_footer_month
-  EditBox 255, 5, 20, 15, renewal_footer_year
-  Text 10, 35, 70, 10, "Resident First Name:"
-  EditBox 80, 30, 70, 15, client
-  Text 165, 35, 65, 10, "Spouse First Name:"
-  EditBox 230, 30, 70, 15, spouse
-  Text 15, 60, 60, 10, "Worker Signature:"
-  EditBox 80, 55, 220, 15, worker_signature
-  CheckBox 65, 80, 125, 15, "Check here to CASE:NOTE actions.", case_note_checkbox
-  ButtonGroup ButtonPressed
-    OkButton 195, 80, 50, 15
-    CancelButton 250, 80, 50, 15
-EndDialog
+
+'Listing button numbers for links
+onesource_URL = 101
+epm_URL = 102
 
 Do
     Do
-        err_msg = ""
-        Dialog Dialog1
-        cancel_without_confirmation
-        IF IsNumeric(maxis_case_number) = false or len(maxis_case_number) > 8 then err_msg = err_msg & vbNewLine & "* Enter a valid case number."
-        If trim(renewal_footer_month) = "" or len(renewal_footer_month) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a two-digit renewal month."
-        If trim(renewal_footer_year) = "" or len(renewal_footer_year) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a two-digit renewal year."
+      err_msg = ""
+
+      Dialog1 = ""
+      BeginDialog Dialog1, 0, 0, 301, 115, "LTC asset transfer dialog"
+        EditBox 75, 5, 50, 15, MAXIS_case_number
+        EditBox 225, 5, 20, 15, renewal_footer_month
+        EditBox 250, 5, 20, 15, renewal_footer_year
+        EditBox 75, 30, 70, 15, client
+        EditBox 225, 30, 70, 15, spouse
+        EditBox 75, 55, 220, 15, worker_signature
+        ButtonGroup ButtonPressed
+          OkButton 190, 95, 50, 15
+          CancelButton 245, 95, 50, 15
+          PushButton 20, 90, 40, 15, "OneSource", onesource_URL
+          PushButton 70, 90, 40, 15, "EPM", epm_URL
+        Text 5, 10, 45, 10, "Case number:"
+        Text 155, 10, 60, 10, "ER date (MM/YY):"
+        Text 5, 35, 70, 10, "Resident First Name:"
+        Text 155, 35, 65, 10, "Spouse First Name:"
+        Text 5, 60, 60, 10, "Worker Signature:"
+        GroupBox 15, 80, 100, 30, "LTC Asset Transfer Policies"
+      EndDialog
+
+      Dialog Dialog1
+      cancel_without_confirmation
+
+      If ButtonPressed > 100 Then
+        err_msg = "Loop"
+        If ButtonPressed = onesource_URL Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-170126"
```

Comment:
Same comment as the Complete List of Scripts (CLoS), I would use this reference: [2.4.2.1.2 MA-LTC Community Spouse Asset Allowance (state.mn.us)](http://hcopub.dhs.state.mn.us/epm/2_4_2_1_2.htm?rhhlterm=12%20community%20spouse&rhsearch=12%20months%20community%20spouse).